### ISSUE TITLE
Group deliveries by building

### ISSUE THREAD

**Issue by davidmegginson** [davidmegginson/food-routes#1]
From Josh:

> Another issue that arose today. Multiple deliveries to the same building. I have been sorting that manually and grouping deliveries to the same building within the same route. For instance, we had 3 deliveries going to an apartment building on St Laurent.

> I don't know the solution here. Is scripts smart enough to figure that out? Could we add a "group these deliveries together" option?

**Comment by jcass120** [comment]
Hey David, 

Again, sorry if this is something we have already discussed. 

The sheet is more or less ready to go. I did not use it for tomorrow's deliveries as I had 4 drivers lined up and the sheet only scheduled two of them. Is there a way to arrange it to just balance the deliveries across all available drivers instead?

**Comment by davidmegginson** [comment]
Yes, we could rewrite the script to maximise rather than minimise the
number of drivers. Not sure if that's worthwhile if we might be moving to
Routific (it's just a check-box item there). What do you think?

D

**Comment by jcass120** [comment]
Yes, that is a good point. Best to wait to see how Routific goes.

Josh
